cli-messaging/src/main/java/com/salesforce/messaging/EventKey.java

```diff
@@ -0,0 +1,59 @@
+package com.salesforce.messaging;
```

Comment:
Basically a copy of `pmd-cataloger/src/main/java/sfdc/sfdx/messaging/EventKey.java`.